spec/test_app/config/application.rb

```diff
@@ -11,6 +12,12 @@ class Application < Rails::Application
     # Settings in config/environments/* take precedence over those specified here.
     # Application configuration should go into files in config/initializers
     # -- all .rb files in that directory are automatically loaded.
+    config.to_prepare do
+      if ENV['VERBOSE']
```

Comment:
this env var thing seems kind a random -- should we just output no matter what? or maybe add something in README about using it?